### PR TITLE
Create 2020-06-01-PFTA—Android_v2.0.4.md

### PR DESCRIPTION
**Pull request by Brock018**
Adds PFTA Android 2.0.4 to release notes

### CODE REVIEW

File: _posts/2020-06-01-PFTA—Android_v2.0.4.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: PFTA—Android v2.0.0
```
Comment (ax-slad) [inline _posts/2020-06-01-PFTA—Android_v2.0.4.md:3]: 2.0.4?

File: _posts/2020-06-01-PFTA—Android_v2.0.4.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+This initial release of the Payment Fusion Terminal Application for Android devices includes the following:
+* Substantial UX uplift to deliver a superior patient payment experience
+* The admin menu is now accessible using a standard password as opposed to a unique per terminal maintenance password in the earlier release. However select advanced menus still need per terminal unique maintenance password
+* Tipping is now supported in the confirm transaction flow
+
+
+#### Bug Fixes:
+
+PFTA_Android_ReleaseNotes_2.0.3
```
Comment (ax-slad) [inline _posts/2020-06-01-PFTA—Android_v2.0.4.md:19]: 2.0.4?

File: _posts/2020-06-01-PFTA—Android_v2.0.4.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+title: PFTA—Android v2.0.0
+tag: PFTA-Android
+author: AxiaMed
+---
+We've created the Payment Fusion Terminal Application to support the addition of PAX A80 and A920 devices to the Payment Fusion platform.
+
+#### Features:
+
+This initial release of the Payment Fusion Terminal Application for Android devices includes the following:
+* Substantial UX uplift to deliver a superior patient payment experience
```
Comment (ax-slad) [inline _posts/2020-06-01-PFTA—Android_v2.0.4.md:12]: Isn't UX uplift already mentioned in 2.0.0 and 2.0.3? Are we

File: _posts/2020-06-01-PFTA—Android_v2.0.4.md (hunk 1 above)
Comment (Brock018) [reply _posts/2020-06-01-PFTA—Android_v2.0.4.md:3]: I made some changes to the original 2.0.0 release notes as well

File: _posts/2020-06-01-PFTA—Android_v2.0.4.md (hunk 3 above)
Comment (Brock018) [reply _posts/2020-06-01-PFTA—Android_v2.0.4.md:12]: this is describing 2.0.0

File: _posts/2020-06-01-PFTA—Android_v2.0.4.md (hunk 2 above)
Comment (Brock018) [reply _posts/2020-06-01-PFTA—Android_v2.0.4.md:19]: this was a change to 2.0.3

**Pull request merged**